developer_manual/common/index.rst

```diff
@@ -0,0 +1,34 @@
+======================
```

Comment:
Please delete this file.

`git rm developer_manual/common/index.rst`

then commit